src/createCodeFileModal.ts

```diff
@@ -55,14 +55,16 @@ export class CreateCodeFileModal extends Modal {
 		submitButton.setButtonText("Create");
 		submitButton.onClick(() => this.complete());
 
-		fileNameInput.inputEl.focus();
+		// fileNameInput.inputEl.focus(); it seems unecessary if the modal is closed
```

Comment:
Why is this unnecessary? When the modal is opened, the filename input should directly be focused so that the user can start typing and completing the form without having to use the mouse.

Reply:
well the filename input is focused anyway. just a detail...